_posts/2017-10-30-lambda-world-conference.md

```diff
@@ -0,0 +1,73 @@
+
+* Ugly code with aliens
+* Techniques to refactor with weapons
+
+It's was a funny and didactic workshop, breaking the walls of concepts as _Lens_, _Prism_ and _Traversal_ (not so complex after the workshop ;)).
+
+I attended another interesting workshop about _Common Lisp_ by **Andrew Lawson**. I still remember how much I enjoyed programming in Lisp at the University, but I didn't know about different implementations of Lisp, as well as the existence of Lisp machines.
+
+I'm in this profession because of Maths, so I had fun with the talk about _Maths for Programming, Fun and Creativity_ by **Dave Gurnell** and **Noel Welsh**.
+
+<a href="https://twitter.com/hhariri" target="_blank">**Hadi Hariri**</a> gave an engaging talk covering _Kotlin 102_. And I could see how <a href="https://twitter.com/raulraja">**Raul Raja**</a>, <a href="https://twitter.com/pacoworks" target="_blank">**Paco Estévez**</a> and <a href="https://twitter.com/vergauwen_simon" target="_blank">**Simon Vergauwen**</a> have brought the best of typed functional programming to Kotlin with their library <a href="http://kategory.io" target="_blank">**Kategory**</a>.
+
+My favourite speaker at the event was <a href="https://twitter.com/_zainabali_" target="_blank">**Zainab Ali**</a>. She talked about _recursion schemes_ in the best pedagogical way, based on the paper <a href="https://maartenfokkinga.github.io/utwente/mmf91m.pdf" target="_blank">_Functional Programming with Bananas, Lenses, Envelopes and Barbed Wire_</a>. She showed how _recursion schemes_ make recursion generic, removing much of the boilerplate associated with it.
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-10-30-lambda-world-conference/hylomorphism.png" alt="Hylomorphism" class="img img-responsive" width="80%"/>
+<small>Slide from <a href="http://slides.com/zainabali_/peeling_the_banana#/" target="_blank">'Peeling the Banana - Recursion schemes from first principles'</a> by Zainab Ali</small>
+</center>
+<br/>
+
+**Jeremy Gibbons**, professor of Computing at the University of Oxford, was the star of the final keynote, showing the relation between the structure of multi-dimensional data and Naperian functors.
+
+Do you know another technical conference which provides its own beer, _functional_ beer?
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-10-30-lambda-world-conference/functional-beer.jpg" alt="Functional Beer" class="img img-responsive"/>
+</center>
+<br/>
+
+Organizers deserve my warmest congratulations, because it was one of the best conferences I've attended. It was the third edition of this awesome event and, despite the high number of attendees, it kept its high quality, the personal attention and its family atmosphere. <a href="https://twitter.com/Jorge__Galindo">**Jorge Galindo**</a> is one of the organizers and his daughter is growing at the same time as the event, getting used to face a huge audience:
```

Comment:
"because it was one of the best conferences I've attended"

Should be:

"because it was one of the best conferences I've ever attended"